fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,48 @@
+.container {
+  margin: 0 auto;
+  background-color: rgb(138, 196, 247);
+}
+
+#title,
+#description {
+  text-align: center;
+}
+
+#description {
+  margin-bottom: 50px;
+}
+
+.input-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.input-description {
+  margin-bottom: 10px;
+}
+
+.center {
+  text-align: center;
+}
+
+.radioButton,
+.checkboxContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.commentsP {
+  text-align: center;
+}
+
+#comments {
+  margin-top: 20px;
+}
+
+.buttonContainer,
+.selectContainer {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
:clap:  Congratulations to give a background color to your page. For better users' experience, stylize and design better your form.

Reply:
Awesome!!! It's always a good idea to show the bits you like about the code/app 👌